src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -2000,6 +2000,15 @@ $K = \\frac{\\varphi}{T}$, where $\\varphi$ is areic heat flow rate and $T$ is t
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Coefficient of heat transfer"@en .
 
+quantitykind:CoefficientOfHeatPerformance
```

Comment:
We already have a `qk:HeatingValue`. How about `quantitykind:CoefficientOfHeatingPerformance` here?

Reply:
CoefficientofHeatPerformance was actually a typo - I intended to just type CoefficientofPerformance, which is the term of art. Coefficient of Performance can also apply to the amount of cooling a system can deliver, so having heating in the name may be confusing. 

It's a dimensionless ratio, so I don't think any units need to be associated other than perhaps UNITLESS.

Reply:
Then `CoefficientOfPerformance` is perfect if it is the term of art. Quantitykinds really can be hard to find if they have funny names.